### ISSUE TITLE
Put all the extension in a tabular format

### ISSUE THREAD

**Issue by ajeetraina** [collabnix/docker-community-extensions#95]
@sivolko Can we put all the extensions in a tabular format?

**Comment by sivolko** [comment]
Yes I can,  I m confused with duplicate one. In last PR  a few were duplicated ?

**Comment by collabnix** [comment]
@sivolko Great work. I see a few of them not in a tabular format. Most of them are single ones.
Let's get them too.

**Comment by sivolko** [comment]
Sure, Tonight will Raise PR

**Comment by sivolko** [comment]
All done !
